spec/rubocop/cop/lint/redundant_require_statement_spec.rb

```diff
@@ -124,6 +126,46 @@
         require 'uri'
       RUBY
     end
+
+    it 'does not register an offense and corrects when using requiring `pp` and using `pretty_inspect`' do
```

Comment:
I'd remove some of the duplication in the message by adding some `context` for "when requiring pp".

Reply:
Sure! This should have been DRY 💦 I've updated it.